specs/fault-dispute-game.md

```diff
@@ -255,12 +255,54 @@ The FDG provides the following interface to manage data loaded to the `PreimageO
 function addLocalData(uint256 _ident, uint256 _execLeafIdx, uint256 _partOffset) external;
 ```
 
-The `addLocalData` function loads parts of a pre-image to VM's `PreimageOracle`. Players use this to ensure pre-image
-parts are available to the VM during a step. Because there are multiple sets of local preimage keys that belong to the
-`FaultDisputeGame` contract due to the ability for players to bisect to any block $n \rarrow n + 1$ state transition
-since the configured genesis, the `_execLeafIdx` parameter enables a search for the starting / disputed outputs to be
-performed such that the contract can write to and reference unique local keys in the `PreimageOracle` for each of these
-$n \rarrow n + 1$ transitions.
+The `addLocalData` function loads local data into the VM's `PreimageOracle`. This data consists of bootstrap data for
+the program. There are multiple sets of local preimage keys that belong to the `FaultDisputeGame` contract due to the
+ability for players to bisect to any block $n \rarrow n + 1$ state transition since the configured genesis, the
+`_execLeafIdx` parameter enables a search for the starting / disputed outputs to be performed such that the contract
+can write to and reference unique local keys in the `PreimageOracle` for each of these $n \rarrow n + 1$ transitions.
```

Comment:
It doesn't look like the rich markdown is picking up this arrow syntax. Should be able to use `&uarr;` instead of `\rarrow`. See: https://github.com/ethereum-optimism/optimism/blob/360b25d39ba3ae143070ff99204b898c6d88ad19/specs/fault-dispute-game.md#preimageoracle-interaction

<img width="917" alt="Screenshot 2024-01-08 at 2 44 27 PM" src="https://github.com/ethereum-optimism/optimism/assets/21288394/b29ffe88-6fc5-4a1b-bc0b-83ee8c68b7c8">